peppol-reporting-backend-redis/src/main/java/com/helper/peppol/reporting/backend/redis/PeppolReportingBackendRedisSPI.java

```diff
@@ -97,7 +99,10 @@ protected JedisPool createJedisPool (@Nonnull final IConfig aConfig)
     }
 
     LOGGER.info ("Using Peppol Reporting Redis at '" + sHost + ":" + nPort + "'");
-    return new JedisPool (sHost, nPort);
+    final String username = aConfig.getAsString(CONFIG_PEPPOL_REPORTING_REDIS_USER);
+    final String password = aConfig.getAsString(CONFIG_PEPPOL_REPORTING_REDIS_PASSWORD);
+
+    return new JedisPool(sHost, nPort, username, password);
```

Comment:
These properties can be null, so we don't need to check if they have text or not.